src/components/shared/transactionsTable/index.js

```diff
@@ -0,0 +1,220 @@
+      case 'senderId':
+      case 'recipientId':
+        return (
+          <div className={`${styles.address}`}>
+            <AccountVisual address={transaction[column.key]} size={32} />
+            <span className={`${styles.addressValue}`}>
+              {isWindowSmall
+                ? transaction[column.key].replace(regex.lskAddressTrunk, '$1...$3')
+                : transaction[column.key]}
+            </span>
+          </div>
+        );
+      case 'amount':
+        return (
+          <React.Fragment>
+            <LiskAmount val={transaction[column.key]} />
+&nbsp;
+            {t('LSK')}
+          </React.Fragment>
+        );
+      case 'fee':
+        return (
+          <IconlessTooltip
+            tooltipContent={<p>{`${t('Type')} ${transaction.type}`}</p>}
+            title={t('Transaction')}
+            className="showOnBottom"
+            tooltipClassName={styles.tooltip}
+          >
+            <div>
+              <LiskAmount val={transaction[column.key]} />
+&nbsp;
+              {t('LSK')}
+            </div>
+          </IconlessTooltip>
+        );
+      case 'timestamp':
+        return <DateTimeFromTimestamp time={transaction[column.key] * 1000} token="BTC" />;
+      case 'confirmations':
+        return (
+          <IconlessTooltip
+            tooltipContent={<p>{`${transaction.confirmations}/101 ${t('Confirmations')}`}</p>}
+            title={transaction.confirmations > 0 ? t('Confirmed') : t('Pending')}
+            className="showOnLeft"
+            tooltipClassName={styles.tooltip}
+          >
+            {transaction.confirmations > 0 ? <Icon name="approved" /> : <Icon name="pending" />}
+          </IconlessTooltip>
+        );
+      default:
+        return transaction[column.key];
+    }
+  }
+
+  // TODO add test when sorting is enabled
+  // istanbul ignore next
+  changeSorting(column) {
+    if (column.isSortingColumn) {
+    // Only changes the caret.
+    // TODO: Implementation of sorting functionality to be done in a separate ticket.
+      this.setState({
+        sortingColumn: column.key,
+        ascendingSorting:
+        column.key === this.state.sortingColumn ? !this.state.ascendingSorting : true,
+      });
+    }
+  }
+
+  // TODO add test when sorting is enabled
+  // istanbul ignore next
+  renderTransactions() {
+    const { sortingColumn } = this.state;
+
+    return this.props.transactions.data.sort((a, b) =>
+      (this.state.ascendingSorting
+        ? b[sortingColumn] - a[sortingColumn]
+        : a[sortingColumn] - b[sortingColumn]));
+  }
+
+  loadMore() {
+    const { transactions } = this.props;
+    transactions.loadData({ offset: transactions.data.length });
+  }
+
+  render() {
+    const {
+      title, transactions, columns, isLoadMoreEnabled, t,
+    } = this.props;
+    const { ascendingSorting } = this.state;
+
+    return (
+      <Box width="full" isLoading={transactions.isLoading}>
+        <Box.Header>
+          <h1>{t(title)}</h1>
```

Comment:
```suggestion
          <h1>{title}</h1>
```
The title has the `t(...)` already when defined, so not needed here again.

Reply:
You're right. I removed the redundant translation.